R/zzz.R

```diff
@@ -0,0 +1,24 @@
+#' Global  i18n config list.
+.i18_config <- list()
+
+#' onLoad
+#'
+#' On package load it updates .i18_config reading yaml file from config.
+#'
+#' @param libname library name
+#' @param pkgname package name
+#'
+#' @importFrom yaml yaml.load_file
+.onLoad <- function(libname, pkgname){
+  .i18_config <<- yaml.load_file(system.file("config.yaml", package = "shiny.i18n"))
```

Comment:
hmm so this config is hardcoded in the package and cannot be changed? or are these just defaults?